tests/kafkatest/services/kafka/kafka.py

```diff
@@ -333,6 +340,12 @@ def start_cmd(self, node):
         return cmd
 
     def start_node(self, node):
+
+        # prevent linux from throwing IOExceptions at scale
+        node.account.ssh("sudo sysctl -w vm.max_map_count=10000000")
```

Comment:
This is Linux-specific (and even potentially Linux-version-specific).  We should not fail the test if this doesn't succeed.  Let's catch the error and log it, but continue, in that case.